src/main/scala/com/databricks/labs/mosaic/core/geometry/point/MosaicPoint.scala

```diff
@@ -3,7 +3,7 @@ package com.databricks.labs.mosaic.core.geometry.point
 import com.databricks.labs.mosaic.core.geometry.MosaicGeometry
 import com.databricks.labs.mosaic.core.MosaicCoreException
 import com.databricks.labs.mosaic.core.geometry.linestring.MosaicLineString
-import com.uber.h3core.util.GeoCoord
+import com.databricks.labs.mosaic.core.types.model.Coordinates
```

Comment:
@edurdevic I like abstraction from external GeoCoord type coming from H3.
We may need to rename some of the methods like getGeoCoords and similar. 
Not critical at all at the moment. Just something to keep in mind.